### PR TITLE
Add Insect Flesh [+ Insect based recipes pt I]

### PR DESCRIPTION
**Pull request by pisskop**
Adds in insect flesh, the ability for critters made of said flesh to drop it when butchered, and adds in basic recipes to use this flesh with.

The insects contain more calcium than other meat, is less appealing, and can now be used for relevant recipes.  Tainted insect meat exists in the cases where it should/could appear.

### CODE REVIEW

**Comment by pisskop** [comment]
Relevant to issue #18625

File: data/json/items/comestibles/carnivore.json [hunk 1]
```diff
@@ -216,6 +253,28 @@
...
+    "name_plural": "insect jerky",
+    "description": "Salted, dried insect meat that never goes bad, but will make you thirsty.",
+    "weight": 40,
+    "container": "bag_plastic",
+    "color": "brown",
+    "charges": 3,
+    "stack_size": 3,
+    "parasites": 0,
+    "healthy": 0,
+    "spoils_in": 0,
```
Comment (Cyrano7) [inline data/json/items/comestibles/carnivore.json]: So you don't need to put in things as 0 that are already defaulted as such. Like "healthy", "spoils_in", or "parasites".

Comment (pisskop) [inline data/json/items/comestibles/carnivore.json]: i do if im editting from a copy where those particular fields are nonzero.

Comment (Cyrano7) [inline data/json/items/comestibles/carnivore.json]: Ah my mistake. I'm still getting use to the copy-from thing.

Comment (Cyrano7) [inline data/json/items/comestibles/carnivore.json]: But there are still already defaulted entries in this PR elsewhere.

File: data/json/recipes/recipe_medsandchemicals.json [hunk 2]
```diff
@@ -1239,8 +1242,7 @@
       [ "mutagen", 1 ]
     ],
     [
-      [ "meat_tainted", 3 ],
-      [ "dry_meat_tainted", 3 ],
+      [ "meat_insect_tainted", 3 ],
```
Comment (Reclusive-reptile) [inline data/json/recipes/recipe_medsandchemicals.json]: How do bug guts make you a squid person? Mullusca and Athropoda are not remotely the same.

Comment (pisskop) [inline data/json/recipes/recipe_medsandchemicals.json]: youre right.  i was uder the impression giant slugs used iflesh

**Comment by mugling** [comment]
Idea is reasonable but lets see if we can write a simpler implementation. I'd like to try and add `TAINTED`, `INSECT` etc flags so we don't require lots of duplicate recipes

File: data/json/recipes/recipe_medsandchemicals.json [hunk 3]
```diff
@@ -679,10 +680,11 @@
...
     ],
     [
-      [ "honey_bottled", 3],
-      [ "honeycomb", 3 ],
+      [ "honey_bottled", 6 ],
+      [ "honeycomb", 6 ],
       [ "bee_sting", 3 ],
-      [ "wasp_sting", 3 ]
+      [ "wasp_sting", 3 ],
+      [ "meat_insect_tainted", 3 ]
```
Comment (Coolthulhu) [inline data/json/recipes/recipe_medsandchemicals.json]: Why tainted?

**Comment by pisskop** [comment]
making item tags sounds well emough, but is above my level of tinkering maybe.

i would imagine this pr would suffice until such tags were made.  these basic recipes make insect meat functional and providing a nutritional resource so basic nutritional testing can occur.

ill look over the copypaste for exraneous zeroes and do you feel strongly about tainted meat being an ingredient in insect serum?

**Comment by pisskop** [comment]
e:

this also allows mods to add the (temp) recipe fillers if they want

**Comment by mugling** [comment]
> these basic recipes make insect meat functional and providing a nutritional resource so basic nutritional testing can occur.

The problem is this yields items that later need migrations

File: data/json/items/comestibles/carnivore.json [hunk 4]
```diff
@@ -109,6 +109,43 @@
...
+    "spoils_in": 24
+  },
+  {
+    "id": "meat_insect_cooked",
+    "copy-from": "meat_insect",
+    "type": "COMESTIBLE",
+    "name": "seared insect meat",
+    "description": "Freshly cooked insect meat.  Very nutritious.",
+    "proportional": { "price": 1.5 },
+    "parasites": 0,
```
Comment (Voskhod-1) [inline data/json/items/comestibles/carnivore.json]: No parasites from eating raw meat?

Comment (pisskop) [inline data/json/items/comestibles/carnivore.json]: according to my pr it has 1.2x as many as meat

Comment (mugling) [inline data/json/items/comestibles/carnivore.json]: looks correct

**Comment by pisskop** [comment]
I am remiss to let this die, but theres not a whole lot left for it.

Why would adding temporary recipes be bad?

**Pull request closed without merging**